src/components/Total.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+
+const Total = props => {
+    const [total, setTotal] = useState('£0.00')
+
+    // USE EFFECTS
+    useEffect(() => {
```

Comment:
great that you found hooks like useEffect to implement for your work - for anyone who has not seen useEffect, tells ell React that your component needs to do something after render